projectRM/projectRM/FirstApp/ViewController.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  ViewController.swift
+//  projectRM
+//
+//  Created by Александр Троицкий on 21.09.2022.
+//
+import Darwin
+import UIKit
+/// firstVC
+class ViewController: UIViewController {
```

Comment:
```suggestion
class ViewController: UIViewController {
```
название дай нормальное